magenta-lib/src/main/scala/magenta/input/DeploymentResolver.scala

```diff
@@ -0,0 +1,78 @@
+package magenta.input
+
+
+object DeploymentResolver {
+  val DEFAULT_REGIONS = List("eu-west-1")
+
+  def resolve(config: RiffRaffDeployConfig): List[Either[(String, String), Deployment]] = {
```

Comment:
A case class? There was some thought about replacing Either with Validated from cats, but possibly a little later once we understand what we are doing a little better.